src/pool/srv_target.c

```diff
@@ -88,6 +88,21 @@ pool_child_lookup_noref(const uuid_t uuid)
 	return NULL;
 }
 
+struct ds_pool_child *
+ds_pool_child_find(const uuid_t uuid)
```

Comment:
given the logic for find and lookup is mostly the same why not just add a bool opt to lookup to return all?

Reply:
that would cause more code changes. (every caller of ds_pool_child_lookup() needs be changed).